pkg/cmd/project/view/view.go

```diff
@@ -0,0 +1,106 @@
+package view
+
+import (
+	"fmt"
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/OctopusDeploy/cli/pkg/constants"
+	"github.com/OctopusDeploy/cli/pkg/factory"
+	"github.com/OctopusDeploy/cli/pkg/output"
+	"github.com/OctopusDeploy/cli/pkg/usage"
+	"github.com/OctopusDeploy/cli/pkg/util/flag"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/client"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/projects"
```

Comment:
> imported and not used: "github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/projects"

@benPearce1 what IDE are you using?

This kind of problem hasn't happened on a PR before because JetBrains GoLand will automatically fixup any imports every time you save the file, and I think VSCode does the same if the extensions are working properly. 

I find GoLand to be nice but Dom+JB prefer VSCode. Both seem to work well. If you would like a hand getting GoLand up and running sing out and I'd be happy to help

Reply:
I believe that this will be fixed when the update to `go-octopusdeploy` is fixed. I was using the following line during development to use an updated version:

`replace github.com/OctopusDeploy/go-octopusdeploy/v2 => ../go-octopusdeploy/`

Reply:
Oh right, makes sense.

I find it helpful in these situations to leave a comment on the PR next to the failing code and say something like "This will be resolved when https://github.com/OctopusDeploy/go-octopusdeploy/pull/145 merges"

Reply:
I called it out, but could have been more explicit, plus I didn't know which specific bit was going to be impacted, still a bit green